manifest/me.art0007i/MessageCopy/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "MessageCopy",
+    "id": "me.art0007i.MessageCopy",
+    "description": "Allows you to copy all messages, not just text ones.",
+    "category": "Dashboard",
+    "sourceLocation": "https://github.com/art0007i/MessageCopy",
```

Comment:
404? might be private

Reply:
should be good now. I forgot to create the repo at all actually oops